infrastructure/variables.tf

```diff
@@ -1,17 +1,26 @@
-variable "product" {}
-
-variable "component" {}
-
+## Defaults
+variable "product" {
+  default = "pip"
+}
+variable "component" {
+  default = "sds"
+}
 variable "location" {
   default = "UK South"
 }
-
 variable "env" {}
-
-variable "subscription" {}
-
-variable "deployment_namespace" {}
-
+variable "subscription" {
+  default = ""
+}
+variable "deployment_namespace" {
+  default = ""
+}
 variable "common_tags" {
-  type = "map"
+  type = map(string)
+}
+variable "team_name" {
+  default = "PIP DevOps"
 }
+variable "team_contact" {
+  default = "#vh-devops"
+}
```

Comment:
what are these here for?

Reply:
Have them in the other infra pipelines, so copied here.
do they get populated by the pipeline?

Reply:
yes https://github.com/hmcts/cnp-jenkins-library/blob/master/vars/spinInfra.groovy#L50

Reply:
removed